src/test/java/org/assertj/vavr/api/MapAssert_containsExactly_Test.java

```diff
@@ -0,0 +1,139 @@
+    }
+
+    @Test
+    void should_pass_if_Map_and_entries_parameter_are_empty() {
+        final Tuple2<String, String>[] entries = new Tuple2[0];
+        assertThat(LinkedHashMap.<String, String>empty()).containsExactly(entries);
+    }
+
+    @Test
+    void should_fail_if_Map_is_not_empty_and_entries_is_an_empty_array() {
+        final Map<String, String> actual = LinkedHashMap.of("key1", "value1", "key3", "value3");
+        final Tuple2<String, String>[] entries = new Tuple2[]{};
+
+        assertThatThrownBy(
+                () -> assertThat(actual).containsExactly(entries)
+        )
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("The array of entries to look for should not be empty");
+    }
+
+    @Test
+    void should_fail_if_entries_parameter_is_null() {
+        assertThatThrownBy(
+                () -> assertThat(LinkedHashMap.<String, String>empty()).containsExactly((Tuple2<String, String>[]) null)
+        )
+                .isInstanceOf(NullPointerException.class)
+                .hasMessage("The array of entries to look for should not be null");
+    }
+
+    @Test
+    void should_fail_if_Map_and_entries_have_different_sizes() {
+        assertThatThrownBy(
+                () -> assertThat(LinkedHashMap.<String, String>empty()).containsExactly(ENTRY1)
+        )
+                .isInstanceOf(AssertionError.class)
+                .hasMessage(
+                        "\nActual and expected should have same size but actual size is:\n" +
+                        " <0>\n" +
+                        "while expected is:\n" +
+                        " <1>\n" +
+                        "Actual was:\n" +
+                        "<LinkedHashMap()>"
+                );
+    }
+
+    @Test
+    void should_fail_if_one_of_entries_is_null() {
+        final Map<String, String> actual = LinkedHashMap.of("key1", "value1", "key3", "value3");
+
+        assertThatThrownBy(
+                () -> assertThat(actual).containsExactly(ENTRY1, null)
+        )
+                .isInstanceOf(NullPointerException.class);
+    }
+
+    @Test
+    void should_fail_when_Map_is_null() {
+        assertThatThrownBy(
+                () -> assertThat((Map<String, String>) null).containsExactly(ENTRY1)
+        )
+                .isInstanceOf(AssertionError.class)
+                .hasMessage(shouldNotBeNull().create());
+    }
+
+    @Test
+    void should_fail_if_Map_does_not_contain_all_entries() {
+        final Map<String, String> actual = LinkedHashMap.of("key1", "value1", "key3", "value3");
+
+        assertThatThrownBy(
+                () -> assertThat(actual).containsExactly(ENTRY1, ENTRY2)
+        )
+                .isInstanceOf(AssertionError.class)
+                .hasMessage(
+                        "\nExpecting:\n" +
+                        "  <LinkedHashMap((key1, value1), (key3, value3))>\n" +
+                        "to contain exactly (and in same order):\n" +
```

Comment:
and in *the* same

Reply:
To change this, we would have to update assertj-core messages of failed assertions. We can make PR to assertj